source/common/event/timer_impl.h

```diff
@@ -23,15 +23,16 @@ class TimerUtils {
    * before doing any conversions. When the passed in duration exceeds INT32_MAX max seconds, the
    * output will be clipped to yield INT32_MAX seconds and 0 microseconds for the
    * output argument. We clip to INT32_MAX to guard against overflowing the timeval structure.
-   * Throws an EnvoyException on negative duration input.
```

Comment:
Can you update the comment with the new behavior on negative duration input, please?